[docs] Add docs for custom dashboards and widgets (#178)
OpenBAS-Platform/docs: docs/usage/scenario.md

To create a scenario, hit the + button on the bottom right of the screen and define general metadata that make sense for you. 

![Create a scenario](./scenario/assets/create-scenario.png)

Once done, the scenario is accessible in the list. Click on it to see its details and define them.

> NB: the dashboard selected will be shown in the Analysis tab but only for the simulations of this scenario.

Reviewer: ```suggestion
NB: The selected dashboard will appear in the Analysis tab, but only for simulations related to this scenario.
```
